src/BlazorWebView/src/SharedSource/StaticContentHotReloadManager.cs

```diff
@@ -86,10 +86,20 @@ private static (string AssemblyName, string RelativePath) GetAssemblyNameAndRela
 			var requestPath = new Uri(requestAbsoluteUri).AbsolutePath.Substring(1);
 			if (ContentUrlRegex.Match(requestPath) is { Success: true } match)
 			{
+				var assemblyName = match.Groups["AssemblyName"].Value;
+				var relativePath = match.Groups["RelativePath"].Value;
+
+				// Remove the fingerprint from scoped CSS bundles, since CSS hot reload will send new content without the fingerprint.
+				// LibraryName.<fingerprint>.bundle.scp.css -> LibraryName.bundle.scp.css
+				if (relativePath.StartsWith($"{assemblyName}.", StringComparison.Ordinal) && relativePath.EndsWith(".bundle.scp.css", StringComparison.Ordinal))
```

Comment:
Is relativePath in the case you're solving guaranteed to be just a filename and not a subpath?

Reply:
Yes, for these bundle files (*.bundle.scp.css) they always exist in `_content/<AssemblyName>/*.bundle.scp.css` without any subpath. This assumption is part of the CSS hot reload code too, when it sends over the newly built bundle file, it knows the destination path. I'll add a comment about it here.